Automattic/jetpack
# Add the WooCommerce installer menu item to wpcom and atomic sites.

Sync changes from D67193-code to Jetpack.

Fixes #

#### Changes proposed in this Pull Request:

* Adds the https://wordpress.com/woocommerce-installation/ menu item to
both atomic and wpcom sites.

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

There should be no change for normal Jetpack sites, this change only affects wpcom and Atomic sites.

Test Atomic site **with** "woop" site sticker applied:
- With WooCommerce installed and active => WooCommerce menu should point at wc-admin
- With Woo installed but not active => WooCommerce menu should point at https://wordpress.com/woocommerce-installation/
- With Woo not installed => WooCommerce menu should point at https://wordpress.com/woocommerce-installation/

Test Atomic site **without** "woop" site sticker applied:
- With WooCommerce installed and active => WooCommerce menu should point at wc-admin
- With Woo installed but not active => Should not see the WooCommerce menu item
- With Woo not installed => Should not see the WooCommerce menu item

#### Before
![Screenshot 2021-09-24 at 15-55-27 My Home ‹ Personal - Simple Site - KO — WordPress com](https://user-images.githubusercontent.com/811776/134625287-1d374281-47f3-47e3-95f2-460404f80a0b.png)

#### After
![Screenshot 2021-09-24 at 15-55-07 My Home ‹ wooptestupgrade — WordPress com](https://user-images.githubusercontent.com/811776/134625278-143dcf04-cd59-489b-a93a-ab78947a2ae6.png)
